src/iperf_api.c

```diff
@@ -953,11 +954,19 @@ iperf_on_connect(struct iperf_test *test)
     struct sockaddr_in *sa_inP;
     struct sockaddr_in6 *sa_in6P;
     socklen_t len;
+    time_t now_secs;
+    unsigned long long now_millisecs;
+
+    iperf_time_now(&now);
+    now_millisecs = (time_t) iperf_time_in_usecs(&now) / 1000;
+    now_secs = (time_t) (now_millisecs / 1000);
+#include <stdio.h>
+    printf("millisecs= %lld ms\n", now_millisecs);
+    printf("secs= %ld s\n", now_secs);
```

Comment:
Is this debugging code?

Reply:
Yikes, sorry! This wasn't supposed to end up in my commit